.travis.yml

```diff
@@ -22,6 +22,8 @@ before_install:
   - pip install future
   - pip install matplotlib
   - pip install netCDF4
+  - pip install beautifulsoup4
+  - pip install lxml
```

Comment:
Do we need to add these to setup.py as well?

Reply:
good catch.  fixing.